### PR TITLE
Fix visual prompting template file name

### PR DESCRIPTION
**Pull request by sungchul1**
### How to test
When using two test cases for `resume` in integration test, weights file cannot be found in the second test.
Test cases were reduced from 2 to 1 for every tests.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: tests/integration/cli/visual_prompting/test_visual_prompting.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import copy
 import pytest
 
-from otx.cli.registry import Registry
+from src.otx.cli.registry import Registry
```
Comment (yunchu) [inline tests/integration/cli/visual_prompting/test_visual_prompting.py]: the module name was not changed so you don't need to put 'src.' to the module importing.

File: src/otx/algorithms/visual_prompting/tasks/__init__.py
Comment (yunchu) [inline src/otx/algorithms/visual_prompting/tasks/__init__.py:1]: FYI. If you want to expose all modules in current namespace, you don't need to define `__all__` in this file. the empty (only has some license statement comments) file is enough.

File: tests/integration/cli/visual_prompting/test_visual_prompting.py (hunk 1 above)
Comment (sungchul1) [reply tests/integration/cli/visual_prompting/test_visual_prompting.py]: Fixed.

File: src/otx/algorithms/visual_prompting/tasks/__init__.py
Comment (sungchul1) [reply src/otx/algorithms/visual_prompting/tasks/__init__.py:1]: Thanks, fixed.

**Review by eunwoosh — APPROVED** [review]

**Review by jaegukhyun — APPROVED** [review]

**Pull request merged**